src/fixtures/viewer/rendering/manifest-with-renderings.json

```diff
@@ -0,0 +1,161 @@
+{
+  "@context": "http://iiif.io/api/presentation/2/context.json",
+  "@type": "sc:Manifest",
+  "@id": "https://figgy.princeton.edu/concern/scanned_resources/f93c3171-43a6-4db3-9692-ba394d669320/manifest",
+  "label": [
+    "Release: Telegram Sent to the Minister of Foreign Affairs in Honduras"
+  ],
+  "viewingHint": "individuals",
+  "metadata": [
+    { "label": "Date Created", "value": ["1956 July 19"] },
+    { "label": "Extent", "value": ["1 box"] },
+    { "label": "Identifier", "value": ["ark:/88435/1g05fh554"] },
+    {
+      "label": "Title",
+      "value": [
+        "Release: Telegram Sent to the Minister of Foreign Affairs in Honduras"
+      ]
+    },
+    { "label": "Language", "value": ["English"] },
+    { "label": "Publisher", "value": ["Dulles, John Foster (1888-1959)"] },
+    { "label": "Container", "value": ["Box 349"] },
+    {
+      "label": "Member Of Collections",
+      "value": ["John Foster Dulles Papers MC016"]
+    }
+  ],
+  "sequences": [
+    {
+      "@type": "sc:Sequence",
+      "@id": "https://figgy.princeton.edu/concern/scanned_resources/f93c3171-43a6-4db3-9692-ba394d669320/manifest/sequence/normal",
+      "rendering": [
+        {
+          "@id": "https://figgy.princeton.edu/catalog/f93c3171-43a6-4db3-9692-ba394d669320/pdf",
+          "label": "Download as PDF",
+          "format": "application/pdf"
+        }
+      ],
```

Comment:
As noted, I don't know what should happen with this when upgraded to v3.

Reply:
@tpendragon Just a heads up on the above comment, the Viewer component uses `vault` to normalize to Pres 3, and this `rendering` location gets lost in translation.

Reply:
I assume we put it here because it's where UV wanted it, if we move books to Clover we'll just move it to the Manifest level.